### PR TITLE
Create Lalrinfeli-Hnamte

### PR DESCRIPTION
**Pull request by Lalrinfeli50**
Added Mentee Details as per Template provided

### CODE REVIEW

**Comment by aku1310** [comment]
Hey @Lalrinfeli50 , please update this to a .md file :)

**Comment by Lalrinfeli50** [comment]
Hi! I just renamed the file with a .md so is that correct? It seems that
the changes I made have been done in another branch called patch-1. I'm not
sure if it's correct so can you please tell me if you can see the pull
request.
Thank You!

**Review by aku1310 — APPROVED** [review]

**Comment by aku1310** [comment]
> Hi! I just renamed the file with a .md so is that correct? It seems that the changes I made have been done in another branch called patch-1. I'm not sure if it's correct so can you please tell me if you can see the pull request. Thank You!

Its fine now! Merging it
